server.js

```diff
@@ -16,6 +16,10 @@ app.get('/search', (req, res) => {
   res.send({});
 });
 
+app.get('/users/:userid', (req, res) => {
```

Comment:
Given that we're going to have to make an async call for user info, I'd map the page itself to `/profile/:id` or something. That way, we can reserve `/users/:id` for the API call.